pkg/config/validation/validation.go

```diff
@@ -3623,3 +3624,28 @@ func validateWasmPluginSHA(plugin *extensions.WasmPlugin) error {
 	}
 	return nil
 }
+
+func validateWasmPluginVmConfig(vm *extensions.VmConfig) error {
+	if vm == nil {
+		return nil
+	}
+
+	keys := sets.NewSet()
+	for _, env := range vm.Env {
+		if env.Name == "" {
+			return fmt.Errorf("invalid env name")
+		}
+
+		if keys.Contains(env.Name) {
+			return fmt.Errorf("duplicate env")
+		}
+		keys.Insert(env.Name)
+
+		if env.ValueFrom == extensions.EnvValueSource_INLINE &&
```

Comment:
Same question here. I think empty string is perfectly valid value.

Reply:
+1